components/edit-collective/Menu.js

```diff
@@ -198,7 +199,7 @@ const sectionsDisplayConditions = {
   [EDIT_COLLECTIVE_SECTIONS.PAYMENT_RECEIPTS]: c => isOneOfTypes(c, ORGANIZATION, USER),
   [EDIT_COLLECTIVE_SECTIONS.VIRTUAL_CARDS]: c =>
     isOneOfTypes(c, COLLECTIVE, FUND) && hasFeature(c.host, FEATURES.PRIVACY_VCC) && c.hasVirtualCards,
-  [EDIT_COLLECTIVE_SECTIONS.TICKETS]: c => isType(c, EVENT),
+  [EDIT_COLLECTIVE_SECTIONS.TICKETS]: c => isType(c, EVENT) && !isPastEvent(c),
```

Comment:
I don't see why this is desirable. Unless this is fixing a clear problem, remove.

Reply:
@znarf : So without this change, the admins will be able to still create tickets for past events. But in this PR, we are hiding the tickets section in the collective for past events. So it didn't make sense to me to show the tickets section in the collective settings but then hide the tickets section. So in essence why would a user be able to create tickets if they cannot even see the tickets section? That's my rational to include this change. Does that make sense? 🤔

Reply:
I understand but as I said, this is not desirable.

It's confusing to remove Edit sections like this, maybe the admins want to see past tickets that were here before? You don't know so it's better to leave it, doesn't hurt anyone.

Reply:
Makes sense. I've went ahead and added it back. 👍🏽